velox/exec/trace/test/QueryTraceTest.cpp

```diff
@@ -202,4 +202,83 @@ TEST_F(QueryTracerTest, traceMetadata) {
     ASSERT_EQ(actualConnectorConfigs.at(key), expectedConnectorConfigs.at(key));
   }
 }
+
+TEST_F(QueryTracerTest, task) {
```

Comment:
Do you have a case with no plan nodes but metadata only? Also can you add a failure test with invalid config? Thanks!

Reply:
>can you add a failure test with invalid config?

Sure, added.

>Do you have a case with no plan nodes but metadata only?

It would throw in `AssertQueryBuilder` if `planNode` is null (no plan nodes). Is this a valid state that a task is creation without any plan nodes?

Reply:
Yes, let's expect an throw there? VELOX_ASSERT_USER_THROW?